### ISSUE TITLE
Message parser: Some messages have out-of-bounds emote tag

### ISSUE THREAD

**Issue by RAnders00** [robotty/dank-twitch-irc#22]
Example that failed in the message parser:

```
@badge-info=subscriber/3;badges=subscriber/3;color=#0000FF;display-name=Linkoping;emotes=25:40-44;flags=17-26:S.6;id=744f9c58-b180-4f46-bd9e-b515b5ef75c1;mod=0;room-id=188442366;subscriber=1;tmi-sent-ts=1566335866017;turbo=0;user-id=91673457;user-type= :linkoping!linkoping@linkoping.tmi.twitch.tv PRIVMSG #queenqarro :Då kan du begära skadestånd och förtal Kappa
```

`emotes=25:40-44` is out of bounds and makes the message parser fail

**Comment by ctlaltdefeat** [comment]
This still happens:
```
@badge-info=;badges=moderator/1,partner/1;color=#5B99FF;display-name=StreamElements;emotes=86:30-39/822112:73-79;flags=22-27:S.5;id=03c3eec9-afd1-4858-a2e0-fccbf6ad8d1a;mod=1;room-id=506590738;subscriber=0;tmi-sent-ts=1588638345928;turbo=0;user-id=100135110;user-type=mod :streamelements!streamelements@streamelements.tmi.twitch.tv PRIVMSG #nobru :╔ACTION A LOJA AINDA NÃO ESTÁ PRONTA BibleThump , AGUARDE... NOVIDADES EM BREVE FortOne╔
```
in this case
```
End index 79 is out of range for given message string
```

Perhaps it's worth allowing the emote parser to fail without failing the entire message parser?

**Comment by johnpyp** [comment]
For the time being, would it be acceptable to just not fill the emotes, but still allow the message to be parsed? I don't think it's a great idea to fail the whole connection because of it.

**Comment by RAnders00** [comment]
fixed in 470d218ade87122c6dc9606f1fd6e8fe75557fdb
